index.html

```diff
@@ -43,104 +51,184 @@ <h2 class="hsecond">I am a Software Developer</h2>
     </section>
     <section class="recent-work">
       <h2 class="title-work">My Recent Work</h2>
```

Comment:
- [ ] As requested by the previous reviewer when we hover on any project the content of the card should be replaced with the `see project` button. Please fix this issue and make sure it matches the design on the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)template.

#### See this image of Figma design, The mouse hovers on the first project and we can see it's showing the see project button and the content of the card has been changed. If you still need more clarification please see this [project](https://devmahmoud.github.io/portfolio/).

![image](https://user-images.githubusercontent.com/83803414/161078036-f0f7012f-31ac-4d0c-9142-0d106895cd93.png)